app/grandchallenge/reader_studies/models.py

```diff
@@ -455,6 +455,8 @@ class Question(UUIDModel):
     )
     order = models.PositiveSmallIntegerField(default=100)
 
+    csv_headers = ["Question text", "Answer type", "Required", "Image port"]
+
```

Comment:
Last question: would it be better to use an OrderedDict for the csv headers (keys) and corresponding values? I'm worried that these 2 lists could get out of sync.

Reply:
Good point, I'll update it.